packages/suite/src/utils/wallet/coinmarket/coinmarketUtils.ts

```diff
@@ -342,6 +342,7 @@ export const coinmarketBuildAccountOptions = ({
     });
 
     const groups: CoinmarketAccountsOptionsGroupProps[] = [];
+    const skipNetworkNativeAccounts: NetworkSymbol[] = ['op', 'base'];
```

Comment:
We are going to miss this when adding new L2s, can you add some smart filter which will find out from `networksConfig`? that symbol and displaySymbol is different?